### ISSUE TITLE
Hurdlr case study: leveraging a brand animation & iphone border color 

### ISSUE THREAD

**Issue by gabrielataginya** [BraveUX/braveux#401]
![Screen Shot 2018-03-22 at 5.28.10 PM.png](https://images.zenhubusercontent.com/59e75702a468affe6df7237a/b96e1261-0d11-44ac-ad27-5ff875ad65a4)

The mobile animation is a bit staticky. 

Also, is there anyway we could use a different cellphone border because it's so similar to the background in the desktop screen.

**Comment by theseangoodwin** [comment]
I agree with the animation comment. @ekfuhrmann maybe we can speed up the desktop screen motion in this composition to end when the mobile device ends? Normally the stagger is nice, but here it just feels slow.

**Comment by theseangoodwin** [comment]
The grey issue will be fixed when we resolve #427
